oca_dependencies.txt

```diff
@@ -1,3 +1,7 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
 
 stock-logistics-warehouse
+account-analytic
```

Comment:
oca_dependancies are no longer necessary, as we should be using pip  for installing